src/hooks/useSession.js

```diff
@@ -1,9 +1,12 @@
-import { useSelector } from 'react-redux';
+import { useSelector, shallowEqual } from 'react-redux';
 
 const useSession = () =>
-  useSelector(({ session }) => ({
-    authenticated: session.authenticated,
-    user: session.user
-  }));
+  useSelector(
+    ({ session }) => ({
```

Comment:
is it possible to do something like:
```suggestion
    ({ session: { authenticated, user } }) => ({
        authenticated,
        user
    }), ...
```
?

Reply:
This looks like nesting if statements, I would prefer not to touch it

Reply:
@kevquincke good idea! I'll do it now

@manuelcheveste I didn't get what you said 😅 it's just destructuring the object

Reply:
tbh, is not that bad like this, but It can become easy to abuse this feature, and then you can end with something like this
```
session: {
  user: { document: { type } }
}
```
and this can go on and on and it will become hard to understand